velox/functions/prestosql/tests/ArithmeticTest.cpp

```diff
@@ -279,6 +279,18 @@ TEST_F(ArithmeticTest, power) {
       "pow(c0, c1)", baseDouble, exponentDouble, expectedDouble);
 }
 
+TEST_F(ArithmeticTest, powerNan) {
```

Comment:
nit: Can you write a test of inf raised to nan ?

Reply:
Done. I confirmed all the results match to Presto.